### ISSUE TITLE
The link to patches is broken

### ISSUE THREAD

**Issue by SSteve** [eclab/flow#4]
The link to patches (https://cs.gmu.edu/~eclab/projects/flow/patches.zip) in readme.md is 404.

**Comment by eclab** [comment]
Looks like it's been removed from the server: but worse, Flow.app.zip seems to be broken too.  I'll build a new Flow version and post it.

**Comment by eclab** [comment]
Okay,  I've posted version 9 of Flow, and uploaded new patches.  MAYBE this will work right?

**Comment by SSteve** [comment]
Yes, the patches link is working now. I downloaded the app earlier today. I did the `sudo spctl` thing (I'm running Mojave 10.14.6) and the app still wouldn't run ("You can’t open the application “Flow” because it may be damaged or incomplete."). I was able to run it by going into the package contents and double-clicking flow.jar. I'll download the new version and let you know what happens. It might be tomorrow, though. My internet is slow and I'm about to leave the computer for the night and start making dinner.

**Comment by SSteve** [comment]
Flow version 9 works fine.
